server/src/main/java/io/druid/server/listener/resource/AbstractListenerHandler.java

```diff
@@ -200,7 +202,7 @@ public final void use_AbstractListenerHandler_instead()
 
   protected abstract
```

Comment:
copy pasted from older code ... but yeah it probably wasn't right... changed in whole class